src/main/java/jenkins/branch/MultiBranchProject.java

```diff
@@ -567,6 +567,11 @@ protected View getWelcomeView() {
         return new MultiBranchProjectEmptyView(this);
     }
 
+    @Override
+    public View getView(String name) {
+        return getPrimaryView();
```

Comment:
I think it would make sense if it was possible to add custom view.

![captura el 2016-04-29 a las 22 16 01](https://cloud.githubusercontent.com/assets/1021745/14928407/1ba5a352-0e58-11e6-810b-778cf858ff2a.png)